test/integration/user_test.rb

```diff
@@ -20,6 +20,16 @@ class UserIntegrationTest < IntegrationTestWithJavascript
     test "login" do
       login_user(users(:admin).login, "secret")
     end
+
+    test "attempt to login disabled user via internal authentication" do
+      login_user(users(:disabled).login, "secret", false)
```

Comment:
We could use `one` here as:
```suggestion
      user(:one).update(disabled: true)
      login_user(users(:one).login, "secret", false)
```